rupture-models/20030821-fiordland/McGinty_Robinson2007/2103645.csv

```diff
@@ -1,101 +1,101 @@
-2,-45.15,166.96,-35231,11987.5,24301.8242,1.92,35,35,95,5000,6000,7.43
-2,-45.15,166.96,-30316.1,15428.9,24301.8242,2.27,35,35,95,5000,6000,5.59
-2,-45.15,166.96,-25401.2,18870.4,24301.8242,2.49,35,35,95,5000,6000,3.91
-2,-45.15,166.96,-20486.3,22311.8,24301.8242,2.6,35,35,95,5000,6000,2.69
-2,-45.15,166.96,-15571.4,25753.3,24301.8242,2.6,35,35,95,5000,6000,2.69
-2,-45.15,166.96,-10656.5,29194.8,24301.8242,2.49,35,35,95,5000,6000,3.91
-2,-45.15,166.96,-5741.57,32636.2,24301.8242,2.27,35,35,95,5000,6000,5.59
-2,-45.15,166.96,-826.655,36077.7,24301.8242,1.92,35,35,95,5000,6000,7.43
-2,-45.15,166.96,4088.26,39519.1,24301.8242,1.27,35,35,95,5000,6000,9.34
-2,-45.15,166.96,-42495.2,11901.1,27169.707,0.78,35,35,95,5000,6000,9.92
-2,-45.15,166.96,-37580.3,15342.5,27169.707,1.65,35,35,95,5000,6000,8.15
-2,-45.15,166.96,-32665.4,18784,27169.707,2.04,35,35,95,5000,6000,6.51
-2,-45.15,166.96,-27750.4,22225.4,27169.707,2.27,35,35,95,5000,6000,5.13
-2,-45.15,166.96,-22835.5,25666.9,27169.707,2.41,35,35,95,5000,6000,4.28
-2,-45.15,166.96,-17920.6,29108.4,27169.707,2.41,35,35,95,5000,6000,4.28
-2,-45.15,166.96,-13005.7,32549.8,27169.707,2.27,35,35,95,5000,6000,5.13
-2,-45.15,166.96,-8090.8,35991.3,27169.707,2.04,35,35,95,5000,6000,6.51
-2,-45.15,166.96,-3175.89,39432.7,27169.707,1.65,35,35,95,5000,6000,8.15
-2,-45.15,166.96,1739.03,42874.2,27169.707,0.78,35,35,95,5000,6000,9.92
-2,-45.15,166.96,-44844.4,15256.1,30037.5898,0.09,35,35,95,5000,6000,10.73
-2,-45.15,166.96,-39929.5,18697.6,30037.5898,0.95,35,35,95,5000,6000,9.11
-2,-45.15,166.96,-35014.6,22139,30037.5898,1.65,35,35,95,5000,6000,7.68
-2,-45.15,166.96,-30099.7,25580.5,30037.5898,1.92,35,35,95,5000,6000,6.56
-2,-45.15,166.96,-25184.8,29021.9,30037.5898,2.1,35,35,95,5000,6000,5.92
-2,-45.15,166.96,-20269.9,32463.4,30037.5898,2.1,35,35,95,5000,6000,5.92
-2,-45.15,166.96,-15354.9,35904.9,30037.5898,1.92,35,35,95,5000,6000,6.56
-2,-45.15,166.96,-10440,39346.3,30037.5898,1.65,35,35,95,5000,6000,7.68
-2,-45.15,166.96,-5525.12,42787.8,30037.5898,0.95,35,35,95,5000,6000,9.11
-2,-45.15,166.96,-610.205,46229.2,30037.5898,0.09,35,35,95,5000,6000,10.73
-2,-45.15,166.96,-47193.6,18611.2,32905.4727,0.09,35,35,95,5000,6000,11.72
-2,-45.15,166.96,-42278.7,22052.6,32905.4727,0.09,35,35,95,5000,6000,10.26
-2,-45.15,166.96,-37363.8,25494.1,32905.4727,0.78,35,35,95,5000,6000,9.01
-2,-45.15,166.96,-32448.9,28935.5,32905.4727,1.27,35,35,95,5000,6000,8.08
-2,-45.15,166.96,-27534,32377,32905.4727,1.5,35,35,95,5000,6000,7.57
-2,-45.15,166.96,-22619.1,35818.5,32905.4727,1.5,35,35,95,5000,6000,7.57
-2,-45.15,166.96,-17704.2,39259.9,32905.4727,1.27,35,35,95,5000,6000,8.08
-2,-45.15,166.96,-12789.3,42701.4,32905.4727,0.78,35,35,95,5000,6000,9.01
-2,-45.15,166.96,-7874.35,46142.8,32905.4727,0.09,35,35,95,5000,6000,10.26
-2,-45.15,166.96,-2959.44,49584.3,32905.4727,0.09,35,35,95,5000,6000,11.72
+SEGMENT,lat,lon,NORTHING,EASTING,DEPTH,SLIP,STRIKE,DIP,RAKE,WIDTH,LENGTH,RUPTURETIME
```

Comment:
I guess it is `RUPTURE TIME` 
a detail in the format header though

Reply:
headers were updated